x-pack/solutions/security/plugins/security_solution/public/data_view_manager/hooks/use_data_view.ts

```diff
@@ -36,7 +36,8 @@ export const useDataView = (
     sourcererAdapterSelector(dataViewManagerScope)
   );
   const newDataViewPickerEnabled = useIsExperimentalFeatureEnabled('newDataViewPickerEnabled');
-  const [localStatus, setLocalStatus] = useState<SharedDataViewSelectionState['status']>('loading');
+  const [localStatus, setLocalStatus] =
+    useState<SharedDataViewSelectionState['status']>('pristine');
```

Comment:
If we set this to pristine, then all the `null` returns make sense, as we don't want to see anything when the dataViews haven't initialized. After initialization, then we can rely on `loading`, `ready`, and `error`